guides/release/testing/testing-components.md

```diff
@@ -29,8 +29,24 @@ This will do the necessary setup for testing your component for you,
 including setting up a way to access the rendered DOM of your component later on in the test,
 and cleaning up once your tests in this module are finished.
 
-```javascript {data-filename="tests/integration/components/pretty-color-test.js"}
-import { module } from 'qunit';
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        If you generated your component using <code>ember generate component pretty-color</code> it will already have generated
+        this file for you with all the boilerplate needed to get started. We are describing the steps to build to that same
+        boilerplate for educational purposes. 
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
+  </div>
+</div>
+
+
```

Comment:
It might be worth noting the file extension -- I keep tripping myself up by creating files with a `.js` extension for tests and expecting `<template>` to work